psydac/api/grid.py

```diff
@@ -100,7 +100,7 @@ def ext(self):
         return self._ext
 #==============================================================================
 class BasisValues():
-    def __init__( self, V, grid, nderiv ):
+    def __init__( self, V, grid, nderiv , ext=None):
```

Comment:
Can you remind me of what `ext` means?

Reply:
the direction of the normal vector in the boundary we use it to determine the boundary